Shellcheck complains:

```
In live/live-root/root/.local/bin/gnome-kiosk-wrapper line 12:
if [[ $(gnome-kiosk --help 2> /dev/null | grep 'enable-vt-switch') ]]; then
   ^-- SC3010 (warning): In POSIX sh, [[ ]] is undefined.
      ^-- SC2143 (style): Use grep -q instead of comparing output with [ -n .. ].

For more information:
  https://www.shellcheck.net/wiki/SC3010 -- In POSIX sh, [[ ]] is undefined.
  https://www.shellcheck.net/wiki/SC2143 -- Use grep -q instead of comparing ...
```

--- a/live/live-root/root/.local/bin/gnome-kiosk-wrapper
+++ b/live/live-root/root/.local/bin/gnome-kiosk-wrapper
@@ -9,7 +9,7 @@ OPTS="";
 
 # We need to invoke gnome-kiosk with option --enable-vt-switch for versions >= 50.
 # Otherwise switching to virtual console will be blocked by default. bsc#1261932
-if [[ $(gnome-kiosk --help 2> /dev/null | grep 'enable-vt-switch') ]]; then
+if gnome-kiosk --help 2> /dev/null | grep -q 'enable-vt-switch'; then
     OPTS="--enable-vt-switch";
 fi;
 

